.github/workflows/Nightly_Perf_Env_CI.yml

```diff
@@ -378,14 +378,13 @@ jobs:
           git checkout main
           git config --global user.email "${{ secrets.EMAIL }}"
           git config --global user.name  "${{ secrets.USER_NAME }}"
+          git config pull.rebase false  # merge (the default strategy)
+
           git add "$GITHUB_WORKSPACE"/benchmark_runner/grafana/perf/dashboard.json
           git commit -m "Update grafana json file"
 
-          # Pull the latest changes from the remote main branch
-          git pull origin main
-
-          # Push the changes to the remote main branch
-          git push origin main
+          git pull https://${{ secrets.GIT_TOKEN }}@${{ secrets.GIT_REPOSITORY}} main
+          git push https://${{ secrets.GIT_TOKEN }}@${{ secrets.GIT_REPOSITORY}} main
```

Comment:
What is the issue here?

Reply:
no special issue, prefer to use it as we use in bumb version: https://github.com/redhat-performance/benchmark-runner/blob/9f3813f983d433adff778a3d5913d1f80ec95d14/.github/workflows/Perf_Env_Build_Test_CI.yml#L284-L297

Reply:
That seems unnecessary; isn't origin already correct?

Reply:
I know that its working.
I can try:

```
git pull main
git push main
```
no need origin, we update the upstream.

Reply:
OK, it's just easier to read without all that extra.